scripts/autodeploy.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Since we use this in a lot of repos, wouldn't it make sense to have this as a github action that we can share everywhere?

Reply:
It could be a good improvement, but I have to research a little bit if there are options to implement a shared mechanism that connects all the repos. I'll take a look